- ros_topic_name: "clock"
  gz_topic_name: "clock"
  ros_type_name: "rosgraph_msgs/msg/Clock"
  gz_type_name: "gz.msgs.Clock"
  direction: GZ_TO_ROS

- ros_topic_name: "/left_force_torque"
  gz_topic_name: "left_force_torque"
  ros_type_name: "geometry_msgs/msg/Wrench"
  gz_type_name: "gz.msgs.Wrench"
  direction: GZ_TO_ROS

- ros_topic_name: "/right_force_torque"
  gz_topic_name: "right_force_torque"
  ros_type_name: "geometry_msgs/msg/Wrench"
  gz_type_name: "gz.msgs.Wrench"
  direction: GZ_TO_ROS
